src/app/loadout/loadout-utils.ts

```diff
@@ -63,6 +67,27 @@ export function getLight(store: DimStore, items: DimItem[]): number {
   }
 }
 
+/** Returns a map of armor hashes to stats. There should be just one of each item */
+export function getArmorStats(
```

Comment:
Does this need the same logic as weapons where it fills in missing items?

Reply:
I thought about it, but wasn't sure if it should be filled with your max or current items. I figured we should just show the armor stats only if you have a full set of armor in the loadout, otherwise it may just be ambiguous to what the stats will actually be when the loadout is applied.